How to install on Arch?

Trying the installation script gives this error

```
2025-12-14 20:19:03 URL:https://raw.githubusercontent.com/wiedehopf/tar1090/master/install.sh [19504/19504] -> "-" [1]
633676 (process ID) old priority 0, new priority 10
bash: line 37: adduser: command not found
bash: line 37: adduser: command not found
[ERROR] Error in line 37 when executing: adduser --system --home-dir "$ipath" --no-create-home tar1090
```

I'd preferably try to install without creating a new user, I just need a tool to view the output by readsb visually

--- a/install.sh
+++ b/install.sh
@@ -34,7 +34,7 @@ mkdir -p "$gpath"
 
 if useSystemd && ! id -u tar1090 &>/dev/null
 then
-    adduser --system --home "$ipath" --no-create-home --quiet tar1090 || adduser --system --home-dir "$ipath" --no-create-home tar1090 || sudo useradd -r -d "$ipath" -M tar1090
+    adduser --system --home "$ipath" --no-create-home --quiet tar1090 || adduser --system --home-dir "$ipath" --no-create-home tar1090 || useradd -r -d "$ipath" -M tar1090
 fi
 
 # terminate with /

